src/docs/verbose-messaging.dox

```diff
@@ -0,0 +1,62 @@
+/*! @page verbose_messaging Verbose Messaging
+
+As part of normal operation, WiredTiger will produce verbose messages
+around various events, actions and possible issues/errors. Verbose
+messages are spread across various categories, representing different sub-systems
+throughout WiredTiger. Verbose messages can be usefully leveraged to monitor,
+debug and diagnose events across WiredTiger's runtime.
+
+@section verbosity_levels Verbosity Levels
+
+WiredTiger associates a verbosity level with each message. The verbosity
+level denotes to the severity of a given message, ranging from high
```

Comment:
I think we can remove the `to` in `denotes to`.

Reply:
Good point, fixed 6657c6e35a876d